### PR TITLE
feat(ui): Change all integration settings view to use `Panel`

### PR DESCRIPTION
**Pull request by billyvg**
* Refactors all project settings views that contain integrations to use
new `Panel` component
* Refactors inactive plugins list
* Moves integration icons to be loaded w/ webpack and out of CSS
* Add integration icon to header for plugin config

image

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by ckj** [comment]
> Add integration icon to header for plugin config

Oooooo that looks good.

File: src/sentry/static/sentry/less/integration-icons.less [hunk 1]
```diff
@@ -1,120 +0,0 @@
-/**
-* Integration icons
```
Comment (ckj) [inline src/sentry/static/sentry/less/integration-icons.less]: 🙌 🙌 🙌

**Review by ckj — APPROVED** [review]
Design-wise, this is 🔥

**Review by MaxBittker — APPROVED** [review]
hard to review this, but what you did with the icons looks good and I'm sure if it doesn't break integration test selectors it's good 👍

**Comment by MaxBittker** [comment]
oops, one snapshot left

**Pull request merged**